web/src/layouts/app-layouts.tsx

```diff
@@ -340,16 +372,16 @@ function AppFooter() {
   const settings = useSettingsContext();
 
   const customFooterContent =
-    settings?.enterpriseSettings?.custom_lower_disclaimer_content;
-
-  // When there's custom footer content, show it
-  if (!customFooterContent) return null;
+    settings?.enterpriseSettings?.custom_lower_disclaimer_content ||
+    "[Onyx v2.0.8-dev](https://www.onyx.app/) - Open Source AI Platform";
```

Comment:
Is this a hardcoded-string? We'll want to have the version number be dynamically rendered, no?

Also, do we even want to default to this string in the first place? Can't we just have it be empty?